lua/entities/gmod_wire_expression2/base/parser.lua

```diff
@@ -1339,14 +1362,27 @@ function Parser:Expr17()
 		if isnumber(tokendata) then
 			return self:Instruction(trace, "num", tokendata)
```

Comment:
This needs to be changed to a literal instruction. This part is the handling of constants.

Reply:
Good spot - fixed, and made sure that nowhere else generates a num[ijk]/str instruction.